### DISCUSSION TITLE
Not able to upload only the image in public repository

### DISCUSSION THREAD

**Discussion by Dinesh0723** [community/community#61335] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am getting error while uploading only the image file in comment section in public repository.
**Error :**  "Something went really wrong, and we can't process that file".

**Comment by nirav-gajera** [comment]
1. Image Corruption: Verify that the image file is not corrupted. Try opening the image in an image viewer or editor to ensure it is working correctly.

2. Clear Cache and Cookies: Sometimes, issues with file uploads can be related to caching or cookies. Try clearing your browser's cache and cookies and then attempt to upload the image again.


3. Try Different Browsers: If you are using a specific browser, try uploading the image using a different browser to see if the issue persists.

4. Check GitHub Status: It's also possible that there might be temporary issues on GitHub's end that are causing the problem. You can check the GitHub Status page to see if there are any ongoing issues.
